Release notes for Keycloak 20 - OpenBanking UK and FIPS (#1709)
keycloak/keycloak-documentation: securing_apps/topics/oidc/fapi-support.adoc

* If your client does not use PAR, make sure that it uses encrypted OIDC request objects. This can be achieved by using a client profile with the `secure-request-object` executor configured with `Encryption Required` enabled.
* Make sure that for JWS, the client uses the `PS256` algorithm. For JWE, the client should use the `RSA-OAEP` with `A256GCM`. This may need to be set in all the link:{adminguide_link}#_oidc_clients[Client Settings] where these algorithms are applicable.

[[_open-banking-uk]]
==== Open Banking UK

{project_name} is compliant with the https://www.openbanking.org.uk/[Open Banking UK]. On top of the <<_fapi-support,FAPI 1 Advanced>>, there is also requirement to
use the `intent` concept to issue tokens.

It works in a way that a client sends an authorization request with this intent. For example, UK OpenBanking requires the following steps:

> 1. A client requests an access token in Client Credentials Grant, and an authorization server issues an access token.

Reviewer: ```suggestion
1. A client retrieves an access token from Keycloak using Client Credentials Grant
```
